DeepSpeech.py

```diff
@@ -1169,6 +1213,10 @@ def _next_epoch(self):
                     # The current epoch should also have a validation part
                     self._epochs_running.append(Epoch(self._epoch, self._num_jobs_dev, set_name='dev', report=is_display_step))
 
+                    # State Machine flag for decision making regarding early stopping in next epoch function call
+                    self._dev = True
```

Comment:
Ah - I see what you mean. But why do we need a "dev" state at all? Why not checking, if early stopping is activated and enough losses got accumulated in the array and then just check - whatever state the state-machine is in.